### DISCUSSION TITLE
Disable missing Google Play Services alert

### DISCUSSION THREAD

**Discussion by gbalduzzi** [invertase/react-native-firebase#4787] [Q&A]
Hi,

I'm using `react-native-firebase` for analytics and messaging. We need to deploy our app to some devices (mainly huawei) that do not have Google Play Services installed. 

**I'm ok with analytics and notifications not actually working on those devices**. 

The problem is, right now the app displays an alert when opened:

> ${App Name} won't run without Google Play Services, which are not supported by your device

After closing the alert, _the app works fine_. Is there a way to just disable (or customize) the alert?

**Comment by mikehardy** [comment]
That's a good question - I'm not actually sure where that is coming from? I know `@react-native-firebase/app` has some services for detecting google play services installation status and responding to it, but I was not aware if it was automatic and/or threw warning messages

https://github.com/invertase/react-native-firebase/pull/3601/files is where these items were added, they may help you instrument / search the code to see maybe where this is coming from?
